rd-net/Test.RdFramework.Generator/ArgsProxyTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Test.RdFramework.Generator;
+
+[Generator]
+public class ArgsProxyTestGenerator : ISourceGenerator
+{
+  public void Execute(GeneratorExecutionContext context)
+  {
+    for (int i = 1; i < 55; i++)
+    {
+      var args = string.Join(", ", Enumerable.Range(0, i).Select(i => i % 2 == 0 ? i.ToString() : $"\"{i}\""));
+      var parameters = string.Join(", ", Enumerable.Range(0, i).Select(i => i % 2 == 0 ? $"int p{i}" : $"string p{i}"));
+      var body = string.Join(" + ", Enumerable.Range(0, i).Select(i => i % 2 == 0 ? $"p{i}" : $"p{i}.Length"));
+      string source = $@"// <auto-generated/>
+using JetBrains.Diagnostics;
+using JetBrains.Rd.Reflection;
+using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+namespace Test.RdFramework.Reflection;
+
+[TestFixture]
+public partial class ProxyGeneratorCalls
+{{
```

Comment:
Note: you can use C# 11 `$$"""` feature to avoid these ugly `{{`, I think 😄

Reply:
I guess this requires update to 7.0

Reply:
Only to SDK 7, but you're right. I'll note that in my backlog.